### PR TITLE
Correct typos and gramma

### PR DESCRIPTION
**Pull request by scriptengine**
I have corrected a couple of typos and some gramma. Nothing added.  The biggest change was the word adding in 
Edit the file adding the code listed below to the end of the file

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/python_script.markdown [hunk 1]
```diff
@@ -37,15 +37,15 @@ It is not possible to use Python imports with this integration. If you want to d
...
 It is created as a first step, to help with:
 
-- Demonstrate how to setup the script
+- Demonstrating how to setup the script
 - How to process the input data
 - How to log the script activity
 - How to troubleshoot / manually call the script.
 
-Start by enabling the Python script and create the first script.
+Start by enabling the Python_script integration and create the first script.
```
Comment (frenck) [inline source/_integrations/python_script.markdown]: This change is incorrect, this is normal language, not code.

```suggestion
Start by enabling the Python Scripts integration and create the first script.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/python_script.markdown (hunk 1 above)
Comment (scriptengine) [reply source/_integrations/python_script.markdown]: frenck, thanks for looking at this, the integration is actually titled "Python Scripts"
https://www.home-assistant.io/integrations/python_script/
I propose:
... the Python **S**cript**s** integration ...
(Changes bolded just for this conversation)

Does that work for you?

Comment (frenck) [reply source/_integrations/python_script.markdown]: Adjusted the suggestion

**Comment by scriptengine** [comment]
Thats great, lets do it

**Comment by scriptengine** [comment]
Reopening as closing was not what I was trying to do.

**Review by frenck — APPROVED** [review]
Thanks, @scriptengine 👍

**Pull request merged**